lib/lndclient/LndClient.ts

```diff
@@ -243,17 +244,23 @@ class LndClient extends SwapClient {
           pending: lnd.getNumPendingChannels(),
         };
         chains = lnd.getChainsList().map(value => value.toObject());
-        blockheight = lnd.getBlockHeight(),
-        uris = lnd.getUrisList(),
+        blockheight = lnd.getBlockHeight();
+        uris = lnd.getUrisList();
         version = lnd.getVersion();
         alias = lnd.getAlias();
+        if (channels.active > 0) {
+          status = SwapClientStatus.Ready;
+        } else {
+          error = errors.LND_IS_UNAVAILABLE(ClientStatus.NotInitialized).message;
```

Comment:
I am not sure whether we should call a client with no clients `NotInitialized`. @kilrau what do you think?

Reply:
Agree, this is not the correct error message. How about  `ClientStatus.NoChannels` ?